test/e2e/loadtest/client/loadtest-config.yaml

```diff
@@ -0,0 +1,3994 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    application: e2e-vegeta
+  name: e2e-vegeta
+  namespace: loadtest-e2e
+data:
+  routes.eskip: |
+    h: PathSubtree("/healthz") -> status(200) -> inlineContent("OK") -> <shunt>;
+    ew: PathSubtree("/ew") -> "http://%TARGET%.ingress.cluster.local/";
+    r: * -> lua(`
+          function request(c)
+              local charset = "0123456789"
+              local s = ''
+              for i = 1, 3 do -- 10^3 different values
+                  local r = math.random(1, #charset)
+                  s = s .. charset:sub(r, r)
+              end
+              c.request.header["x-loadtest"] = s
+          end
+          `) -> "https://%TARGET%.%ZONE%";
+  ew-get.txt: |
+    GET http://127.0.0.1:9090/ew
+  get-long.txt: |
+          GET http://127.0.0.1:9090/ei7choo2eexaph8mahboo5shi6eeyo9phaishae3Naibei6Siiwu9Fu8Rabie5Zo0exoo3aetheeNaiPohvoo8jein9eNgo6neemaef4nohcheag0Pheitheif4fe6Me7xuyo2ENg1sa2ahgheikaeWookoe1dae0nee5aifaeyiis3jahvieru7ooDaetoh6Oi9aeNg0uo6ahsa5Lahzae6chapifeedeiPh0kiagh4iethukainu1cahsahh7fu7ahJ7ahT1thee3hohCeeMei4Un2aiyaijiangularo9cae2Higheit1voPhae7EejuPuYe7ohPie9Shophiphu8oopheP2IeCh9eu7oKiengeit0Tha8Ieshahc8pae6ahnaih0caisieL0tugh5och2hica8ahQui9yeis0ZaiBish2aghooneirohxaeghoshoamiegeiZolee5oPh9Ahmeecaep5Ohv1yeepo4osh8iwiequeiloh1shubaij2Fi1sha6jaijiquaeghiem0koochiach1eiRacha4aijeigou3iemah0eghic6Pahfeepeilee3je3iev6ieL8ALaehao0goocaemier7biaheeMoh9AGedaequ2chieThohshoh7Ungee2ahH6oWaiph7eiluodohTee5uthaegaiCi8aequ6xa8zahheisaDo9eakeLoo5than6opeiMeih7aetoox4aub8uShaeB2oveiloadtestfucaigegh5ohnaeteo1sine8EthohTauj3xierah8aiseikoDoofe6ahW5Coh3aisaeZ7wohda3xai6ohnee3uw9auroh8yo2Ip4eiku3TauW4Oong0ungeimoi0loochi2eiche3eeShech2uiz0aehee5WugheilaeSiV0zohdeK3Ahv7Yahn4ooch6Mah7uu0tu4oog8Dei1eeJie1aekaef9ze3Phaich4yie4uo3iyaeS3huangeeNgei0weesooyie8Oon
+  get.txt: |
```

Comment:
I think load test uses only `get.txt`. Lets remove everything else (and blobs especially) - we can add it anytime when needed

Reply:
done